liferay-workspace/settings.gradle

```diff
@@ -13,4 +13,4 @@ buildscript {
 
 apply plugin: "com.liferay.workspace"
 
-include 'tests:blade.sample.tests', 'tests:blade.servicebuilder.test'
+include 'tests:blade.sample.tests', 'tests:blade.servicebuilder.test'
```

Comment:
we don't need this.  Once you pass in `-P liferay.workspace.modules.dir=modules,tests` we won't need to include these two project includes.